extensions/cql/standard/schema/examples/example07.txt

```diff
@@ -1,3 +1,3 @@
-EXAMPLE 07: Comparison and spatial predicates in a single expression
+EXAMPLE: An example of comparison and spatial predicates in a single expression.
 
-filter=eo_instruments LIKE 'OLI*' AND footprint INTERSECTS POLYGON((43.5845 -79.5442, 43.6079 -79.4893, 43.5677 -79.4632, 43.6129 -79.3925, 43.6223 -79.3238, 43.6576 -79.3163, 43.7945 -79.1178, 43.8144 -79.1542, 43.8555 -79.1714, 43.7509 -79.6390, 43.5845 -79.5442))
+eo_instruments LIKE 'OLI*' AND footprint INTERSECTS POLYGON((43.5845 -79.5442, 43.6079 -79.4893, 43.5677 -79.4632, 43.6129 -79.3925, 43.6223 -79.3238, 43.6576 -79.3163, 43.7945 -79.1178, 43.8144 -79.1542, 43.8555 -79.1714, 43.7509 -79.6390, 43.5845 -79.5442))
```

Comment:
Same comment on the LIKE as above.

Also, why has `filter=` been removed here, but not in other examples? Is there a pattern?